src/Pages/BookDetails/Aside.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+function Aside() {
+  const [save, setSave] = useState(false);
```

Comment:
```suggestion
  const [isSaved, setIsSaved] = useState(false);
```
boolean 타입의 변수를 표현하기 위해 이름을 이와 같이 수정해주면 좋겠네요

Reply:
아 네 감사합니다 !!